config/locales/en.yml

```diff
@@ -152,6 +152,20 @@ en:
           friend at %{invitation_page}.
           Cheers,
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      abuse: "contact our Abuse Team"
```

Comment:
Can I suggest clearer, more distinct keys for lines 159 and 161, e.g. `contact_abuse` for this and `signature` for 161?